cloudflare/resource_cloudflare_custom_ssl.go

```diff
@@ -0,0 +1,284 @@
+package cloudflare
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/pkg/errors"
+)
+
+func resourceCloudflareCustomSsl() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareCustomSslCreate,
+		Read:   resourceCloudflareCustomSslRead,
+		Update: resourceCloudflareCustomSslUpdate,
+		Delete: resourceCloudflareCustomSslDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareCustomSslImport,
+		},
+
+		Schema: map[string]*schema.Schema{
```

Comment:
We're missing some fields from the schema, is this intentional? The API shows the following also available:

- type
- geo_restrictions

Reply:
Will add these

Reply:
These are not supported by v4api https://github.com/cloudflare/cloudflare-go/blob/master/ssl.go#L40-L44. Ill cut a PR against that repo and then update this one likely in a separate PR.

Reply:
🙇 thanks for getting that updated! PR is at cloudflare/cloudflare-go#327